github/ci/prow-deploy/kustom/base/configs/current/orgs/orgs.yaml

```
@@ -659,6 +661,13 @@ orgs:
           - aglitke
         repos:
           containerized-data-importer-api: admin
+      community-health-files:
+        description: Default Community health files.
+        maintainers:
+          - dhiller
+          - brianmcarey
+        repos:
```

We should add @aburdenthehand (our [community facilitator](https://github.com/kubevirt/community/blob/main/MAINTAINERS.md)) here also

```suggestion
          - aburdenthehand
        repos:
```

---

Okay.